src/OnboardingSPA/components/Drawer/stylesheet.scss

```diff
@@ -228,11 +229,31 @@
 
 /*COLOR VARIABLES*/
 $main-color-dark: var(--nfd-onboarding-dark);
-$main-color-light: var(--nfd-onboarding-white);
+$main-color-light: var(--nfd-onboarding-light);
 $main-border-light: var(--nfd-onboarding-border);
 $main-border-main: var(--nfd-onboarding-primary-alt);
 $main-border-main--rgb: var(--nfd-onboarding-highlighted--rgb);
 
+.theme-styles-preview--drawer__list__item__live-preview-container .live-preview__container-custom {
```

Comment:
Hey, the changes look good to me! 
However, since some of these changes are from another PR, let's merge that first ot the trunk and then this.

Reply:
Ig this should be tested first.